changelog.json

```diff
@@ -1,5 +1,22 @@
 {
   "releases": {
+    "1.6.0": [
+      "[New] Help users add their first repo during onboarding - #6474",
+      "[New] \"No local changes\" view makes it easy to find and accomplish common actions - #6445",
+      "[Improved] Repository list when no repositories found - #5566 #6474",
```

Comment:
I think this needs to be moved down the list alongside the other `[Improved]` - while the app does group and sort, I don't believe the marketing site does any grouping of the items.